test/e2e_federation/service.go

```diff
@@ -102,15 +103,15 @@ var _ = framework.KubeDescribe("Federated Services [Feature:Federation]", func()
 
 				if service != nil {
 					By(fmt.Sprintf("Deleting service shards and their provider resources in underlying clusters for service %q in namespace %q", service.Name, nsName))
-					cleanupServiceShardsAndProviderResources(nsName, service, clusters)
+					cleanupServiceShardsAndProviderResources(nsName, service.Name, clusters)
```

Comment:
Ok, I done this modification since the only "service *api.Service" information retrieve in the "cleanupServiceShardsAndProviderResources()" method was its name.

But sure, I will rollback those changes in order to minimalise the PR's footprint .